tests/rados/test_osd_ecpool_inconsistency_scenario.py

```diff
@@ -175,6 +176,8 @@ def run(ceph_cluster, **kw):
         except Exception as e:
             log.info(e)
             return 1
+        msg_obj_count = f"Scenario-2:The object count value is-{obj_count}"
```

Comment:
Line no 164. We should not remove the `osd_scrub_auto_repair` param. it should be set to true itelf.